package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "matrix-public-archive",
+  "name": "matrix-viewer",
   "version": "0.1.0",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
-    "url": "https://github.com/matrix-org/matrix-public-archive"
+    "url": "https://github.com/matrix-org/matrix-viewer"
   },
```

Comment:
CI failure: https://github.com/matrix-org/matrix-public-archive/actions/runs/5558130289/jobs/10152911632#step:4:8
```
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/@opentelemetry/instrumentation-nestjs-core/-/instrumentation-nestjs-core-0.32.3.tgz - Not found
npm ERR! 404 
npm ERR! 404  '@opentelemetry/instrumentation-nestjs-core@https://registry.npmjs.org/@opentelemetry/instrumentation-nestjs-core/-/instrumentation-nestjs-core-0.32.3.tgz' is not in this registry.
npm ERR! 404 
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.
```

Reply:
Seems like a npm problem because `@opentelemetry/instrumentation-nestjs-core@0.32.3` still exists on npm: https://www.npmjs.com/package/@opentelemetry/instrumentation-nestjs-core/v/0.33.0?activeTab=versions

Reply:
Their status site claims all green: https://status.npmjs.org/

But I see plenty of other people also reporting 404 errors on Twitter: https://twitter.com/search?q=%40npmjs&src=typed_query